releasenotes/notes/orchestrator-tag-9e9a7da025799f3a.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Introduce `kube_cluster_name` and `ecs_cluster_name` tags in addition to `cluster_name`.
```

Comment:
can you also add the new option to the changelog?

Reply:
done